src/main/java/org/terasology/scenario/internal/systems/RegionSystem.java

```diff
@@ -90,7 +90,7 @@ public void onAttackEntity(AttackEvent event, EntityRef targetEntity, org.teraso
                     } else {
                         if (!pos.equals(create.firstHit)) {
                             RegionLocationComponent loc = editedRegion.getComponent(RegionLocationComponent.class);
-                            loc.region = Region3i.createBounded(pos, create.firstHit);
+                            loc.region = new BlockRegion(pos, create.firstHit);//Region3i.createBounded(pos, create.firstHit);
```

Comment:
I think the equivalent to `createBounded` is to build the union of those two positions.
```suggestion
                            loc.region = new BlockRegion(pos).union(create.firstHit);
```